docusign-connector/pom.xml

```diff
@@ -41,6 +41,10 @@
         <artifactId>project-build-plugin</artifactId>
         <version>10.0.1</version>
         <extensions>true</extensions>
+        <configuration>
+          <ivyVersion>10.0.3</ivyVersion>
```

Comment:
This is why we should always release a new version of the project-build-plugin when make a release of the platform.

Reply:
mhh, yes you are right -> I add a task for this to my story.